src/components/molecules/Profile/Profile.module.scss

```
@@ -0,0 +1,10 @@
+.wrapper {
+  display: flex;
+  gap: 10px;
+}
+
+.inWrapper {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
```

그때 mixin.scss에 만들어놓은 툴에 flex-direction만 추가해도 좋을 것 같네용

---

```suggestion
  display: flex;
  flex-direction: column;
```

이렇게 말씀하시는 게 맞을까요?

---

아니요 저런 케이스가 있을 경우 mixin.scss flexCenter 만들어진 속성을 쓰고 flex-direction: column을 추가하면 되지 않을까 하는 말이었어요